lib/facilities/mental_health_client.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Facilities
+  class MentalHealthClient < SQL52Client
+    def download
+      @client.execute(select_all_mental_sql_query).to_a
+    end
+
+    private
+
+    def select_all_mental_sql_query
+      "SELECT * FROM #{mental_health_table} ORDER BY #{order_by_column} ASC"
+    end
+
+    def database_name
+      Settings.sql_52.facilities_mental_health.database_name
```

Comment:
Will this (and the table name) change from environment to environment?

Reply:
No, it will be the same in every environment.